website/public/tidal-drum-machines-alias.json

```diff
@@ -0,0 +1,69 @@
+{
+  "AJKPercusyn": "Percysyn",
+  "AkaiLinn": "Linn",
+  "AkaiMPC60": "MPC60",
+  "AkaiXR10": "XR10",
+  "AlesisHR16": "HR16",
+  "AlesisSR16": "SR16",
+  "BossDR110": "DR110",
+  "BossDR220": "DR220",
+  "BossDR55": "DR55",
+  "BossDR550": "DR550",
+  "CasioRZ1": "RZ1",
+  "CasioSK1": "SK1",
+  "CasioVL1": "VL1",
+  "DoepferMS404": "MS404",
+  "EmuDrumulator": "Drumulator",
+  "EmuSP12": "SP12",
+  "KorgDDM110": "DDM110",
+  "KorgKPR77": "KPR77",
+  "KorgKR55": "KR55",
+  "KorgKRZ": "KRZ",
+  "KorgM1": "M1",
+  "KorgMinipops": "Minipops",
+  "KorgPoly800": "Poly800",
+  "KorgT3": "T3",
+  "Linn9000": "9000",
+  "LinnLM1": "LM1",
+  "LinnLM2": "LM2",
+  "MoogConcertMateMG1": "ConcertMateMG1",
+  "OberheimDMX": "DMX",
+  "RhodesPolaris": "Polaris",
+  "RhythmAce": "Ace",
+  "RolandCompurhythm1000": "Compurhythm1000",
+  "RolandCompurhythm78": "Compurhythm78",
+  "RolandCompurhythm8000": "Compurhythm8000",
+  "RolandD110": "D110",
+  "RolandD70": "D70",
+  "RolandDDR30": "DDR30",
+  "RolandJD990": "JD990",
+  "RolandMC202": "MC202",
+  "RolandMC303": "MC303",
+  "RolandMT32": "MT32",
+  "RolandR8": "R8",
+  "RolandS50": "S50",
+  "RolandSH09": "SH09",
+  "RolandSystem100": "System100",
+  "RolandTR505": "TR505",
+  "RolandTR606": "TR606",
+  "RolandTR626": "TR626",
+  "RolandTR707": "TR707",
+  "RolandTR727": "TR727",
+  "RolandTR808": "TR808",
+  "RolandTR909": "TR909",
```

Comment:
kinda wanna add "909" as another alias but its a slippery slope

Reply:
yeah i guess 2 more or less characters doesnt make a lot of difference..